test/PrintAsObjC/classes.swift

```diff
@@ -318,6 +318,13 @@ typealias AliasForNSRect = NSRect
   func testBridgingOptionality(_ a: UnsafePointer<Int>?, b: UnsafeMutablePointer<Int>!, c: AutoreleasingUnsafeMutablePointer<Methods?>?) {}
 }
 
+// CHECK-LABEL: IB_DESIGNABLE
+// CHECK-LABEL: @interface MyDesignableObject : NSObject
```

Comment:
(CHECK-NEXT does handle partial matches, so you don't actually have to include the mangled name.)